ChuckNorrisFactsTests/View Model Tests/ListFactsViewModelTests.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  ListFactViewModel.swift
+//  ChuckNorrisFactsTests
+//
+//  Created by Munir Wanis on 23/05/18.
+//  Copyright © 2018 Wanis. All rights reserved.
+//
+
+@testable import ChuckNorrisFacts
+import Nimble
+import Quick
+import RxSwift
+import RxTest
+
+class ListFactsViewModelTests: QuickSpec {
```

Comment:
Não acha que dessa forma dificuldade bastante a legibilidade?

Reply:
Acho muito pelo contrário!

Reply:
Um problema que eu vejo de fazer assim é a impossibilidade de rodar um teste por vez...

Reply:
Como a gente estava discutindo aqui pessoalmente outro dia (vou colocar aqui só pra ficar registrado), eu resolvi usar ao máximo o framework de `Spec` mais pra conhecer e ver o que da pra fazer. 

Não pretendo usar no dia-a-dia porque sabemos o inferno que é (enquanto o Swift não for API-stable) quando rolam updates da linguagem e as coisas começam a quebrar. Eu ainda prefiro usar o `XCTest` porque é o default da Apple e funciona.

Como eu mexi com Ruby uma época pra mim é muito clara a escrita dos testes com o `Spec`, mas entendo que seja meio ruim pra galera que já tá acostumada com o `XCTest`.